modules/gapi/include/opencv2/gapi/gstreaming.hpp

```diff
@@ -371,6 +371,32 @@ class GAPI_EXPORTS_W_SIMPLE GStreamingCompiled
 };
 /** @} */
 
+namespace gapi {
+namespace streaming {
+/**
+ * @brief Ask G-API to set specific queue capacity for streaming execution.
+ *
+ * For streaming execution G-API has a special concurrent bounded queue to
+ * fetch next frame while current is being processed. This compilation argument
```

Comment:
It is not only about fetching the next frame. Overall this text provides very specific details with no proper context. Nobody knows how it works inside.

I'd just mention that in the streaming mode the pipeline steps are connected with queues and this method controls every queue's size, not mentioning it is bounded, etc. etc.

Reply:
Done